index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <!--  Hello word, We are softwares developers!
+    
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <h1 id="Header" class="flex-container">
+      <img id ="header-img" src="https://i.annihil.us/u/prod/marvel/i/mg/c/80/4d5c2e7075ded/clean.jpg" alt="logo" width="100" height="70">logo
+      <nav id="nav-bar">
+            <a class="nav_link" href="#">cat</a>
+            <a class="nav_link" href="#">Dog</a>
+            <a class="nav_link" href="#">Pig</a>  
+      </nav>
+    </h1>
```

Comment:
You have a beautiful design. However, you will agree with me that using <header> tag to wrap h1 and nav tags is semantically more beautiful.